content/company/values.md

```diff
@@ -2,7 +2,7 @@
 
 These values are the beliefs and principles that help us achieve our [goals](goals/index.md) and [strategy](../company/strategy/index.md) and build an [inclusive team](../communication/code_of_conduct.md). These apply at every level of our organization—"you" refers to individuals, teams, and Sourcegraph as a whole.
 
-- [**Be customer driven**](#be-customer-driven): You proactively work on the right things by continuously orienting your goals around delivering value to your customers.
+- [**Be customer driven**](#be-customer-driven): You earn and keep the trust of our customers by putting their interests first (ahead of Sourcegraph's interests and your team's interests).
```

Comment:
```suggestion
- [**Be customer-driven**](#be-customer-driven): You earn and keep the trust of our customers by putting their interests first.
```

**suggest (non-blocking)**: Adding the part around putting customer interest ahead of Sourcegraph's interest in the main value tagline doesn't feel right. It does feel right to have it in the descriptive text of the value as we can provide better context as to what we mean with this.